tests/src/reloadUnitTest/kotlin/tests/statics.kt

```diff
@@ -35,3 +36,33 @@ fun `test - change top level property`() {
     )
     assertEquals("bar", topLevelProperty)
 }
+
+private fun changeInitializer() {
+    assertEquals("foo", topLevelProperty2)
+    compileAndReload(
+        """
+        package tests
+        object StaticsObject {
+            val property = "bar"
+        }
+    """.trimIndent()
+    )
+}
+
+private fun setStaticsReinitializeMode(mode: StaticsReinitializeMode) {
+    System.setProperty("compose.reload.staticsReinitializeMode", mode.name)
```

Comment:
Let's use the HotReloadProperty directly here :)

Reply:
Done